oracle-linux-image-tools/README.md

```diff
@@ -70,10 +89,13 @@ Files are processed in that order.
 As user you should only make changes in your local `env.properties` where you can override any definition from the previous property files.  
```

Comment:
I would rewrite this to "Changes should be made to a local `env.properties` file which can override any definition made in an upstream property file."